spec/system/candidate_interface/candidate_account_spec.rb

```diff
@@ -44,6 +44,12 @@ def when_i_visit_the_signup_page
     visit '/'
 
     click_on t('application_form.begin_button')
+
+    page.find('#candidate-interface-eligibility-form-eligible-citizen-yes-field').click
+    page.find('#candidate-interface-eligibility-form-eligible-qualifications-yes-field').click
+    page.find('#candidate-interface-eligibility-form-eligible-providers-yes-field').click
```

Comment:
I believe we don't need `page`.

Reply:
Non-blocking: instead of `find` could we use `choose` as this implies that these are radio buttons.

Reply:
Update to use `find`! `choose` will raise:

>      Capybara::ElementNotFound:
>       Unable to find radio button "#candidate-interface-eligibility-form-eligible-citizen-yes-field" that is not disabled
>       Note: It appears you may be passing a CSS selector or XPath expression rather than a locator. Please see the documentation for acceptable locator values.